docs/source/examples/array-numpy.rst

```diff
@@ -0,0 +1,13 @@
+Creating Dask arrays from NumPy arrays
+======================================
+
+We can create Dask arrays from NumPy arrays using ``da.from_array``.
+
+.. code-block:: python
+
+    >>> import numpy as np
+    >>> import dask.array as da
+    >>> x = np.arange(1000)
+    >>> y = da.from_array(x, chunks=(100))
+    >>> y.mean().compute()
+    499.5
```

Comment:
I'm concerned that people will actually do this, which would be a mistake.  If we have this example we should include a pretty strong disclaimer along with it saying that this is a bad idea.